lms/static/scripts/frontend_apps/components/dashboard/LastSyncIndicator.tsx

```diff
@@ -0,0 +1,52 @@
+import type { IconComponent } from '@hypothesis/frontend-shared';
+import { formatDateTime } from '@hypothesis/frontend-shared';
+import classnames from 'classnames';
+import type { JSX } from 'preact';
+import { useMemo } from 'preact/hooks';
+
+import RelativeTime from '../RelativeTime';
+
+export type LastSyncIndicatorProps = Omit<
+  JSX.HTMLAttributes<HTMLDivElement>,
+  'icon' | 'className' | 'dateTime'
```

Comment:
The props type accepts arbitrary HTML props but does not pass them to the `div`. I would prefer to simplify the type if we don't need the customizability yet, especially since this component is not a "presentational" component which wraps a specific HTML element.